From 357d802b6960e5cd9e97752992dd7a8831dd746c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?V=C3=ADt=20Zikmund?=
 <75443448+vit-zikmund@users.noreply.github.com>
Date: Wed, 22 Feb 2023 08:27:17 +0100
Subject: [PATCH] hwdb: fix swapped buttons for Logitech Lift left

(cherry picked from commit 81cfea95e51c72da6765b517e7038b3e7e3dec9f)

Related: RHEL-5950
---
 hwdb.d/60-keyboard.hwdb | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index 10dde4dbec..6ab1468052 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -1328,6 +1328,11 @@ evdev:input:b0003v046DpC309*
  KEYBOARD_KEY_c01b6=images                              # My Pictures (F11)
  KEYBOARD_KEY_c01b7=audio                               # My Music (F12)
 
+# Logitech USB Receiver Mouse (Lift for Business, left-handed)
+# This left-handed ergonomic mouse swaps the left/right buttons
+evdev:input:b0003v046DpC548e0111*
+ KEYBOARD_KEY_90001=btn_right
+ KEYBOARD_KEY_90002=btn_left
 
 ###########################################################
 # Maxdata
